docs/team/shared-infrastructure.md

```diff
@@ -20,8 +20,33 @@ We have [a DockerHub organization](https://hub.docker.com/r/jupyterhub/jupyterhu
 BinderHub and JupyterHub images are pushed to DockerHub as that still works, though we are considering publishing these images to use quay.io as well.
 If you do not have access to this organization and require it, ask a team member to add you.
 
-## Google Cloud Project
+## `mybinder.org` infrastructure
+
+The `mybinder.org` service has a few pieces related to the domain that are described below:
+
+### Ownership of the `mybinder.org` domain
+
+The `mybinder.org` domain is owned by Chris Holdgraf ([@choldgraf](https://github.com/choldgraf)).
+The registrar for this domain is [BlueHost](https://bluehost.com).
+The email address with contact information for the domain is `binder-team@googlegroups.com`.
+
+```{admonition} TODO
+In the future we wish to transfer ownership of this domain away from Chris's personal account.
+However, it is unclear what is the best way to do this right now, and so we are waiting for clarification from the Jupyter project at-large about how the project itself can own mybinder.org instead of Chris. We also plan to use a contact email for this domain that has fewer people on it than the entire team, but are using `binder-team@googlegroups.com` as a stop-gap measure for now to increase our bus factor.
+```
+
+### DNS entries for `mybinder.org`
+
+The DNS for `mybinder.org` is handled by [CloudFlare](https://www.cloudflare.com/), by a central Jupyter account.
+
+(shared:gcp)=
+### Google Cloud Project
 
 The Binder Team uses a Google Cloud Project called `binderhub`.
 This project runs the BinderHub deployment that is found at `gke.mybinder.org`.
 The project is currently funded from a grant from Google.
```

Comment:
I think it's fine if we reveal the explicit details about the gcp project id and gcp organization id. If so, then this is related I think.

```suggestion
### Google Cloud Platform (GCP) Project

The Binder Team uses a GCP project with id [`binderhub-288415`](https://console.cloud.google.com/home/dashboard?project=binderhub-288415) to host `gke.mybinder.org` and top level federation services at `mybinder.org`. The `binderhub-288415` GCP project resides in the GCP organization [`jupyter.org`](https://console.cloud.google.com/iam-admin/iam?organizationId=920697752286).

The `binderhub-288415` GCP project is currently funded by a grant from Google, as represented as GCP credits. The credits belong to billing account belonging to the GCP organization `jupyter.org`, as can be seen [under its billing section](https://console.cloud.google.com/billing?organizationId=920697752286).
```